zerver/lib/topic.py

```diff
@@ -195,6 +189,7 @@ def update_messages_for_topic_edit(
 
     for message in messages_list:
         update_edit_history(message, last_edit_time, edit_history_event)
+        message.last_edit_time = last_edit_time
```

Comment:
Huh.  Why do we end up needing this?

Reply:
Good catch. I missed that when doing the rebase.